experimental/content-type-screen-effect/README.md

```diff
@@ -0,0 +1,92 @@
+# Screen effect content type
+
+This experimental package provides a content type to support adding screen effects to messages.
+
+## What’s a screen effect?
+
+A screen effect is something that happens on a particular trigger, such as sending balloons along with a happy birthday message or sending snowflakes to accompany a holiday message.
+
+## Why screen effects?
+
+Providing screen effects can be a way to surprise and delight an app's users, by presenting them with an unexpected visual effect.
+
+## Install the package
+
+```bash
+# npm
+npm i @xmtp/experimental-content-type-screen-effect
```

Comment:
I kind of always imagined that it would be an experimental content type package that imported all the experimental types not each one individually. 🤔 Do web developers like smaller more granular npm packages like this?

Reply:
hmm, i would think we'd want these to match the same setup as the other non-experimental content types where it's individual (e.g. npm i @xmtp/content-type-reaction). i assume devs would probably just want to import the package as-needed when they want to incorporate that specific type.

WDYT @rygine?

Reply:
yep, smaller packages are generally preferred for browsers